protobuf.sh

```diff
@@ -6,7 +6,7 @@ build_requires:
  - "GCC-Toolchain:(?!osx)"
 prefer_system: "(?!slc5)"
 prefer_system_check: |
-  printf "#include \"google/protobuf/message.h\"\nint main(){}" | c++ -I$(brew --prefix protobuf)/include -Wno-deprecated-declarations -xc++ - -o /dev/null
```

Comment:
Hi, I would use `protoc -h` (which brings up a help screen and exits with 0) instead of `type protoc`. Can you do this change?